tests/tests/users.test.js

```diff
@@ -0,0 +1,68 @@
+const request = require("supertest");
+const axios = require("axios");
+
+const { createApp } = require("../../app");
+const dbDataSource = require("../../api/models/dataSource");
+
+describe("Sign Up", () => {
+  let app;
+
+  beforeAll(async () => {
+    app = createApp();
+    await dbDataSource.initialize();
+  });
+
+  afterAll(async () => {
+    await dbDataSource.query(`SET FOREIGN_KEY_CHECKS=0`);
+    await dbDataSource.query(`TRUNCATE users`);
+    await dbDataSource.query(`ALTER TABLE users AUTO_INCREMENT = 1`);
+    await dbDataSource.query(`SET FOREIGN_KEY_CHECKS=1`);
+    await dbDataSource.destroy();
+  });
+
+  test("FAILED : INVAILD KAKAO TOKEN", async () => {
+    await request(app).post("/users/kakao/signIn");
+    expect(400);
+    expect({ message: "INVALID_KAKAO_ACCESS_TOKEN" });
+  });
+
+  test("SUCCESS : SUCCESS KAKAO SIGNUP AND LOGIN", async () => {
+    axios.get = jest.fn().mockReturnValue({
+      data: {
+        id: 1234,
+        properties: {
+          nickname: "테스트",
+        },
+        kakao_account: {
+          age_range: "15~19",
+          email: "testtest222@gmail.com",
+        },
+      },
+    });
+    const response = await request(app)
+      .post("/users/kakao/signIn")
+      .set({ Authorization: "TEST ACCESS TOKEN" });
+    expect(response.status).toEqual(200);
+    expect(response.body).toHaveProperty("accessToken");
+  });
+
+  test("SUCCESS : SUCCESS KAKAO LOGIN", async () => {
+    axios.get = jest.fn().mockReturnValue({
+      data: {
+        id: 1234,
+        properties: {
+          nickname: "테스트",
+        },
+        kakao_account: {
+          age_range: "15~19",
+          email: "testtest222@gmail.com",
+        },
+      },
+    });
+    const response = await request(app)
+      .post("/users/kakao/signIn")
+      .set({ Authorization: "TEST ACCESS TOKEN" });
+    expect(response.status).toEqual(201);
```

Comment:
```suggestion
    expect(response.status).toEqual(200);
```
테스트 코드도 api 코드 수정에 따라 상태코드 맞춰서 변경해주시고, 승태님 PR이 먼저 병합됨에 따라 발생한 conflict 까지 해결 부탁드릴게요~~~